packages/frontend/core/src/modules/explorer/views/sections/tags/index.tsx

```diff
@@ -28,13 +34,16 @@ export const ExplorerTags = () => {
       tagService.randomTagColor()
     );
     setCreatedTag(newTags);
+    setCollapsed(false);
   }, [t, tagService]);
 
   return (
-    <div className={styles.container}>
+    <Collapsible.Root className={styles.container} open={!collapsed}>
       <CategoryDivider
         className={styles.draggedOverHighlight}
         label={t['com.affine.rootAppSidebar.tags']()}
+        setCollapsed={setCollapsed}
+        collapsed={collapsed}
```

Comment:
Better to extract `Collapsible` & `CategoryDivider` into a layout, and manage states like `collapsed` inside. Since there are many such usage in this PR.

Reply:
I will make them into a common component later